.travis.yml

```diff
@@ -43,3 +52,13 @@ script:
   - ./configure
   - make
   - make test
+  - rm -Rf docs/dev
+  - mv docs/build docs/dev
+  
+deploy:
+  provider: pages                                                                                                                   
+  skip_cleanup: true
+  github_token: $GITHUB_TOKEN
+  on:
+    branch: gh-pages
+  target_branch: gh-pages
```

Comment:
This is going to expose the docs at http://gearman.org/gearmand/dev/docs ... is that what we want? I think it's fine, but the current link is to `/dev/docs` not `/gearmand/dev/docs` so that would need to be made as a follow up commit to the main website.

Reply:
I just tried to make the least effort change, because old docs might be removed in some major release. Those docs were already out there, so I didn't want to break anyone's workflow. If major contributors would agree (and keep promise ;) ) not to write technical docs of the project in a main repository, then fine, we can move sources.

The second challege is the way sources are compiled. AFAIK default jekyll setup and sphinx build of technical docs are mutually exclusive. We would have to set up our own flow for jekyll (if that's true, that might not pay off yet).